x/stakeibc/client/cli/tx_add_validator.go

```diff
@@ -0,0 +1,54 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/Stride-Labs/stride/x/stakeibc/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cast"
+	"github.com/spf13/cobra"
+)
+
+var _ = strconv.Itoa(0)
+
+func CmdAddValidator() *cobra.Command {
```

Comment:
makes sense to expose this for testing -- should we remove this for testnet and mainnet?

Reply:
or restrict to our account as caller, please add TODO for that

Reply:
added a TODO for this! and we already can only add validators that haven't been added